examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -354,6 +354,8 @@ annotations:
   deployment: {}
   # Annotations for each Pod in the Deployment
   pod: {}
+  # Annotations for the Secret (ignored when disabled)
```

Comment:
```suggestion
  # Annotations for the Secret (has no effect when `joinTokenSecret.create` is false)
```

Reply:
fixed though this comment was taken from `extraLabels.secret`